admin/client/src/styles/legacy/_style.scss

```diff
@@ -154,11 +154,16 @@ body.cms {
 	width: 100%;
 }
 
+
 // We have a faux three column layout when displaying Page content in the CMS.
 .has-panel {
 	.cms-tree-view-sidebar,
 	.cms-versions-form {
-		padding: 0 $grid-x*1.5 $grid-y*1.5 $grid-x*2;
+		padding: 0 $grid-gutter-width-half $grid-gutter-width-half;
+	}
+
+	.cms-versions-form {
```

Comment:
yeah i guess it would be best to split the styles, I'll try to get to it today but stuck in the middle of something right now.